src/Uno.UI/UI/Xaml/FontFamilyHelper.wasm.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml;
+
+public partial class FontFamilyHelper
+{
+	/// <summary>
+	/// Pre-loads a font to minimize loading time and prevent potential text re-layouts.
+	/// </summary>
+	/// <returns>True is the font loaded successfuly, otherwise false.</returns>
+	public static Task<bool> Preload(FontFamily family)
```

Comment:
I suggest renaming this to `PreloadAsync` so that it is clear that it can be awaited (also to stay in line with the WinUI/UWP naming convention). Also, the docs don't include the `await` yet, maybe it should be added there too.

Reply:
@MartinZikmund There is analyzer to enforce that. Is it interesting to enable it for the repo?

https://github.com/microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD200.md

Reply:
I don't think we'll be able to enable it for the whole repro (many APIs need to stay as-is to avoid breaking changes), but it may be interesting in other cases, like tools.